TF_Service_dotNet/TouchFree/Interactions/InteractionModules/AirPushInteraction.cs

```diff
@@ -72,17 +72,17 @@ public class AirPushInteraction : InteractionModule
         {
             interactionTuning = _interactionTuning?.Value;
 
-            if (interactionTuning?.AirPushSettings != null)
+            if (_configManager.InteractionConfig?.AirPush != null)
             {
-                speedMin = interactionTuning.AirPushSettings.SpeedMin;
-                speedMax = interactionTuning.AirPushSettings.SpeedMax;
-                distAtSpeedMinMm = interactionTuning.AirPushSettings.DistAtSpeedMinMm;
-                distAtSpeedMaxMm = interactionTuning.AirPushSettings.DistAtSpeedMaxMm;
-                horizontalDecayDistMm = interactionTuning.AirPushSettings.HorizontalDecayDistMm;
-                thetaOne = interactionTuning.AirPushSettings.ThetaOne;
-                thetaTwo = interactionTuning.AirPushSettings.ThetaTwo;
-                unclickThreshold = interactionTuning.AirPushSettings.UnclickThreshold;
-                unclickThresholdDrag = interactionTuning.AirPushSettings.UnclickThresholdDrag;
+                speedMin = _configManager.InteractionConfig.AirPush.SpeedMin;
+                speedMax = _configManager.InteractionConfig.AirPush.SpeedMax;
+                distAtSpeedMinMm = _configManager.InteractionConfig.AirPush.DistAtSpeedMinMm;
+                distAtSpeedMaxMm = _configManager.InteractionConfig.AirPush.DistAtSpeedMaxMm;
+                horizontalDecayDistMm = _configManager.InteractionConfig.AirPush.HorizontalDecayDistMm;
+                thetaOne = _configManager.InteractionConfig.AirPush.ThetaOne;
+                thetaTwo = _configManager.InteractionConfig.AirPush.ThetaTwo;
+                unclickThreshold = _configManager.InteractionConfig.AirPush.UnclickThreshold;
+                unclickThresholdDrag = _configManager.InteractionConfig.AirPush.UnclickThresholdDrag;
             }
```

Comment:
These changes need adding to the "OnInteractionSettingsUpdated" method as well as otherwise they only update on service restart

Reply:
Should be sorted now :+1: